Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 # Update StackStorm version here to rely on other Docker images tags
 appVersion: 3.0dev
 name: stackstorm-ha
-version: 0.10.0
+version: 0.8.4
```

Comment:
I don't know if the mismatch is because of my mistake or is it normal here

Reply:
> @rapittdev As I see, the github diff still shows old content that was already contributed to upstream `master` branch. We can't open an all-writeable branch for community per Github limitations.
> But if you want to open a new PR instead of this one, - that would be fine too.

I prefer to preserve the commit history and the flow of conversation. I will give it some time to clean it up tonight.

Reply:
Sounds good :+1: